src/main/kotlin/carracing/CarRace.kt

```diff
@@ -0,0 +1,25 @@
+package carracing
+
+import kotlin.random.Random
+
+class CarRace(val numCars: Int, val numRounds: Int) {
+    val cars = generateCars()
+
+    fun startRace() {
+        require(cars.size >= MIN_NUMBER_OF_CARS) { "Not enough cars to race" }
+        require(numRounds > MIN_NUMBER_OF_ROUNDS) { "Select at least one round to start the race" }
```

Comment:
Using `require` for validation is a great choice!

If the user input is invalid but only checked when the race starts, it’s a bit late to catch the issue.
It’s usually better to validate as early as possible. 

What do you think about performing the validation during object creation?

Here's a link that might help:
- [[Kotlin Docs] Classes - Constructors and initializer blocks](https://kotlinlang.org/docs/classes.html#constructors)